enhancements/api-review/add-new-CRD-ImageDigestMirrorSet-and-ImageTagMirrorSet-to-config.openshift.io.md

```diff
@@ -326,6 +326,30 @@ Phase 2:
 - components listed in the above [document](https://docs.google.com/document/d/11FJPpIYAQLj5EcYiJtbi_bNkAcJa2hCLV63WvoDsrcQ/edit?usp=sharing) migrate to new APIs ImageDigestMirrorSet, ImageTagMirrorSet
 - register and expose the CRDs.
 
+#### Implementation updates
+Actual implementation for migration path of Phase 2 was different from the original design:
+
+The webhook is not implementable. It cannot convert between different kind of resources. 
+
+If copy existing ImageContentSourcePolicy content to ImageDigestMirrorSet allow both kind of objecting existing, this meeting [notes](https://docs.google.com/document/d/13h6IJn8wlzXdiPMvCWlMEHOXXqEZ9_GYOl02Wldb3z8/edit?usp=sharing) raised the concern that in some cases the final `registries.conf` will lead the customer implying only one of the object is honored.
+
+For the above reason the actual implementation for migration path:
+- implement the migration path:
+  - [openshift/kubernetes#1310](https://github.com/openshift/kubernetes/pull/1310) prohibit coexistence of ICSP and IDMS objects, or ICSP and ITMS objects.
+  - [openshift/oc#1238](https://github.com/openshift/oc/pull/1238) implemented oc command that can convert ImageContentSourcePolicy yaml to ImageDigestMirrorSet yaml. The command can be used by customer when they want to migrate from ICSP to IDMS resources.
+
+#### Update the implementation regarding the customer requests
+[RHIBMCS-117](https://issues.redhat.com/browse/RHIBMCS-117?focusedId=22980820&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22980820) requests both ICSP and IDMS objects exist in the multi-tenant use case, and during the period of migration.
+
+Migration plan:
+1. oc command can generate IDMS yaml files.
+2. creates IDMS object without removing ICSP.
```

Comment:
Explain that this will result in same registries.conf and hence not result in a reboot.